package.json

```diff
@@ -52,7 +52,7 @@
         "precustom-element-json": "lerna exec --ignore \"{@spectrum-web-components/{base,bundle,iconset,modal,shared,opacity-checkerboard,styles,custom-vars-viewer,reactive-controllers,eslint-plugin},stylelint-header,@swc-react/*,documentation,example-project-rollup,example-project-webpack,swc-templates,@types/swc}\" -- rm custom-elements.json ||:",
         "preeleventy": "yarn docs:analyze",
         "prelerna-publish": "rimraf react && yarn get-ready && yarn custom-element-json && yarn build:confirm",
-        "prepare": "husky install",
+        "prepare": "husky",
```

Comment:
Being it _mostly_ worked before, I wonder if there will be any other publish time issues that were otherwise not encountered. This should likely be merged, along with https://github.com/adobe/spectrum-web-components/pull/4385, right before the next release to ensure it goes smoothly and the changes can be backed out of, if needed.

To support that you could even merge these two together and release on a non-`main` branch to get a confirmation before moving to `main`.

Reply:
Yes creating a non `main` release seems to be a safe bet to make sure this change doesnt cause any build failures. Will do once these PRs are landed.